pkg/daemon/ceph/client/filesystem.go

```diff
@@ -195,7 +195,7 @@ func SetNumMDSRanks(context *clusterd.Context, cephVersion cephver.CephVersion,
 	// Now check the error to see if we can even determine whether we should reduce or not
 	if errAtStart != nil {
 		return fmt.Errorf(`failed to get filesystem %s info needed to ensure mds rank can be changed correctly,
-if Ceph version is Luminous (12.y.z) and num active mdses (max_mds) was lowered, USER should deactivate extra active mdses manually: %v`,
+if num active mdses (max_mds) was lowered, USER should deactivate extra active mdses manually: %v`,
```

Comment:
```suggestion
if num active mdses (max_mds) was lowered, USER should deactivate extra active mdses manually: %+v`,
```

I know it's not part of your change, but while we are at it...